processing/src/main/java/org/apache/druid/segment/FloatDimensionIndexer.java

```diff
@@ -52,6 +52,16 @@ public EncodedKeyComponent<Float> processRowValsToUnsortedEncodedKeyComponent(@N
     return new EncodedKeyComponent<>(f, Float.BYTES);
```

Comment:
For this method override, just return `processRowValsToUnsortedEncodedKeyComponent(dimValues, reportParseExceptions, null)` to avoid duplication of code